### PR TITLE
Rename -init hook to -start hook

### PR DESCRIPTION
**Pull request by omron93**
SSIA

@praiskup @pkubatrh @ficap Did I miss something?

### CODE REVIEW

**Comment by praiskup** [comment]
It seems to match the postgresql-container case, so looks OK to me.

**Comment by omron93** [comment]
@praiskup We decided that -init is run before -start? I can't remember...

File: 2.6/root/usr/share/container-scripts/mongodb/README.md [hunk 1]
```diff
@@ -179,6 +179,11 @@ started
 these scripts are skipped if `run-mongod-replication` is run with already
 initialized data directory
 
+##### `mongodb-start/`
+
+same sematics as `mongodb-init/`, except that these scripts are always source
```
Comment (praiskup) [inline 2.6/root/usr/share/container-scripts/mongodb/README.md]: sourced

**Comment by praiskup** [comment]
I don't know whether we made some final decision, @ficap did the research, .. but
if `-init` is supposed to run only once, and `-start` always -- it would be nice to have
consistent feeling in `-start` hook (users can expect that the database is initialized).

Btw., I would prefer double-quotes around all the shell variables in the added code
at least.

File: 2.6/root/usr/share/container-scripts/mongodb/README.md (hunk 1 above)
Comment (omron93) [reply 2.6/root/usr/share/container-scripts/mongodb/README.md]: thanks

**Comment by omron93** [comment]
> I don't know whether we made some final decision, @ficap did the research, .. but
if -init is supposed to run only once, and -start always -- it would be nice to have
consistent feeling in -start hook (users can expect that the database is initialized).

+1

> Btw., I would prefer double-quotes around all the shell variables in the added code
at least.

Where it's missing?

**Comment by omron93** [comment]
Rebased.

[test-openshift]

**Comment by omron93** [comment]
[test-openshift]

**Comment by omron93** [comment]
Rebased to include fix for Fedora images.
[test-openshift]

**Comment by omron93** [comment]
[test-openshift]

**Comment by omron93** [comment]
Tests are passing. No big changes after last review (rebases mainly). So merging.

**Pull request merged**